### ISSUE TITLE
Split Accessibility class

### ISSUE THREAD

**Issue by kblok** [hardkoded/puppeteer-sharp#2532]
Get the Accessibility class ready for the bidi protocol

**Comment by kblok** [comment]
Accessibility doesn't have a base class
